src/main/java/com/codeborne/selenide/HiddenString.java

```diff
@@ -0,0 +1,22 @@
+package com.codeborne.selenide;
+
+public class HiddenString {
+
+  private final String plainTextValue;
+  private final String toStringValue;
+
+  public HiddenString(String plainTextValue, String toStringValue) {
+    this.plainTextValue = plainTextValue;
+    this.toStringValue = toStringValue;
+  }
+
+  public String getPlainTextValue() {
+    return plainTextValue;
+  }
+
+  @Override
+  public String toString() {
+    return toStringValue;
```

Comment:
Since you're promote to mask password with asterisks why do we need to pass plain and tostring value. Just pass orignal value and in toString method just replace chars with asterisks. 

Something like that:
```
public class SensitiveValue {
    private final String plainValue;

    SensitiveData(final String plainValue) {
        this.plainValue = plainValue;
    }

    @Override
    public String toString() {
        return StringUtils.repeat("*", plainValue.length());
    }
}
```

Reply:
@rosolko make sense. But it will be nice to have an ability to change asterisks with custom placeholder

Reply:
We have use case,
where we will store in toString value path in HashiCorp Vault and crypted value from HashiCorp Vault.

This way we can make reports public and users,
that need to debug, decrypt the hidden values from the report know the Vault path and crypted value from the logs or reports and they can decrypt the value if they have permissions to this path in Vault.

So we need to keep it up to the user, what toString value he wants to set.

Reply:
I have updated initial PR comment > Proposed changes with use case and example use.